frontend/src/components/Banner.js

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2021 Christoph Ehm <christoph.ehmendoerfer@campus.tu-berlin.de>
+
+import React, { useEffect, useState } from "react";
+import HeaderTypography from "./HeaderTypography";
+import ParagraphTypography from "./ParagraphTypography";
+import { Link } from "react-router-dom";
+//import appwriteApi from "../api/appwriteApi";
+import RoundedEdgesButton from "./RoundedEdgesButton";
+import { Grid } from "@mui/material"
+
+import { textColor, signupButtonColor } from "../assets/jss/colorPalette";
+import bannerImage from "../assets/img/banner.png"
+
+const backgroundImageStyle = {
+	backgroundImage: `url(${bannerImage})`,
+	backgroundPosition: "center",
+};
```

Comment:
```suggestion
const backgroundImageStyle = {
	backgroundImage: `url(${bannerImage})`,
	backgroundPosition: "center",
	backgroundSize: "cover"
};
```
to have the image always cover the whole div and not repeat itself. Also works great for responsive designs.

Reply:
Thank you a lot, I use this.